analyzer/src/main/java/nl/tudelft/ArtifactId.java

```diff
@@ -0,0 +1,31 @@
+package nl.tudelft;
+
+import java.util.Objects;
+
+public class ArtifactId extends PackageId {
```

Comment:
We might as well just use Maven Artifact objects rather than having a functionally identical clone.

Reply:
I was thinking that as well, but wanted to keep things somewhat backwards compatible. I can change that